testdata/p4_16_samples_outputs/pna-direction.p4.spec

```diff
@@ -75,16 +75,13 @@ apply {
 	shl m.left_shift_tmp m.pna_pre_input_metadata_input_port
 	mov m.pna_pre_input_metadata_direction m.reg_read_tmp
 	and m.pna_pre_input_metadata_direction m.left_shift_tmp
+	shr m.pna_pre_input_metadata_direction m.pna_pre_input_metadata_input_port
+	equ m.pna_pre_input_metadata_direction 0x0
```

Comment:
I think "equ" instruction is not a valid instruction. Just a placeholder kind of instruction added in def file.
Please verify the output with DPDK pipeline to confirm.
If this is invalid, add jumps to assign the right value to direction metadata